packages/pds/src/api/com/atproto/repo/applyWrites.ts

```diff
@@ -29,6 +29,9 @@ export default function (server: Server, ctx: AppContext) {
           'Unvalidated writes are not yet supported.',
         )
       }
+      if (tx.writes.length > 200) {
+        throw new InvalidRequestError('Two many writes. Max: 200')
+      }
```

Comment:
I assume this check lives here because you don't want it to be a hard limit in the lexicon (i.e. for all PDS implementations), is that right?

Reply:
Yea exactly. There are cases where you may want to do more than 200 writes & i feel that we shouldn't put this at the lexicon level. There's a good chance we'll remove this check at some point when we have a better handle on the repercussions of large mutations